README.md

```diff
@@ -2,6 +2,8 @@
 
 # ospd-netstat
 
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/greenbone/PROJECT/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/greenbone/PROJECT/?branch=master)
```

Comment:
`PROJECT` needs to be replaced with `ospd-netstat` for the badge to function properly.

Reply:
sure. i must have committed too quickly. should be good now. make sure to squash commits when merging.